### PR TITLE
Fix #6

### LINKED ISSUE #6: test_that_wikidata_works_with_multiple_brands_one_invalid failing
```
======================================================================
FAIL: test_that_wikidata_works_with_multiple_brands_one_invalid (test_wikimedia_link_issue_reporter.Tests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mateusz/Documents/install_moje/OSM_software/wikibrain_py_package_published/test_wikimedia_link_issue_reporter.py", line 490, in test_that_wikidata_works_with_multiple_brands_one_invalid
    self.assertIsNone(should_be_fine)
AssertionError: <wikibrain.wikimedia_link_issue_reporter.ErrorReport object at 0x7f24aa898d90> is not None
```

Given that https://www.wikidata.org/wiki/Q7501155 is actually defunct, it should not be None, right? And if one would be invalid then error should be still emitted?

@KasperFranz 

from #5

### PR DESCRIPTION
**Pull request by KasperFranz**
This PR fixes the test case for multiple_brands.

Fixes #6

### CODE REVIEW

File: test_wikimedia_link_issue_reporter.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 import wikibrain.wikimedia_link_issue_reporter
 import wikibrain
 import wikimedia_connection.wikimedia_connection as wikimedia_connection
-import wikimedia_connection.wikidata_processing as wikidata_processing
```
Comment (matkoniecz) [inline test_wikimedia_link_issue_reporter.py:7]: wait, it was not needed?

Comment (KasperFranz) [reply test_wikimedia_link_issue_reporter.py:7]: No, it is showing as not used in my editor

Comment (matkoniecz) [reply test_wikimedia_link_issue_reporter.py:7]: and you have run tests in general, right?

Comment (KasperFranz) [reply test_wikimedia_link_issue_reporter.py:7]: Yes of course, it is only used in the test_wikidata_structure

**Comment by matkoniecz** [comment]
Thanks!

**Pull request merged**